minecraftbot/main.ts

```diff
@@ -1,9 +1,14 @@
-const axios = require('axios');
```

Comment:
please do not remove the axios require statement

Reply:
solved fix